identity-apps-core/apps/recovery-portal/src/main/webapp/includes/cookie-consent-banner.jsp

```diff
@@ -0,0 +1,166 @@
+<%--
+~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+~
+~ This software is the property of WSO2 Inc. and its suppliers, if any.
+~ Dissemination of any information or reproduction of any material contained
+~ herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~ You may not alter or remove any copyright or other notice from copies of this content."
+--%>
+
+<%@ page import="org.apache.commons.text.StringEscapeUtils" %>
+<%@ page import="org.apache.commons.lang.StringUtils" %>
+<%@ page import="org.wso2.carbon.identity.mgt.endpoint.util.IdentityManagementEndpointUtil" %>
+
+<%-- Include tenant context --%>
+<jsp:directive.include file="../tenant-resolve.jsp"/>
+
+<%-- Branding Preferences --%>
+<jsp:directive.include file="./branding-preferences.jsp"/>
+
+<%-- Localization --%>
+<jsp:directive.include file="./localize.jsp" />
+
+<%
+    String isCookieproEnabled = System.getenv("is_cookiepro_enabled");
+    if (isCookieproEnabled == null) {
+        isCookieproEnabled = "false";
+    }
+    String cookiepro_domain_script_id = System.getenv("cookiepro_domain_script_id");
+    String initialScriptType = (Boolean.TRUE.toString()).equals(isCookieproEnabled) ? "text/plain" : "text/javascript";
+%>
+
+<%
+    if (!StringUtils.isBlank(cookiePolicyURL) && 
+        ((Boolean.FALSE.toString()).equals(isCookieproEnabled) || 
+        ((Boolean.TRUE.toString()).equals(isCookieproEnabled) && !StringUtils.equals(tenantForTheming, IdentityManagementEndpointConstants.SUPER_TENANT)))) {
+%>
+<div id="cookie-consent-banner" data-testid="cookie-consent-banner" class="ui segment cookie-consent-banner inverted aligned-right hidden transition">
+    <div class="banner-image" data-testid="cookie-consent-banner-image">
+        <div class="theme-icon floated-left hover-rounded tiny transparent cookie-icon" data-testid="generic-icon">
+            <svg class="icon" width="14" height="14" viewBox="0 0 514.374 514.374">
```

Comment:
If we are going to show the old cookie banner for non-super tenant apps, don't we need to load this logo through branding preferences?

Reply:
No. Currently, It is not configurable from the branding section.